scripts/R/manipulate_mead_projected.R

```diff
@@ -251,20 +251,64 @@ add_lines <- function(g_id, data, form.factor, language){
 }
 
 
-read_mead_projected <- function(){
-  
+read_LEGACY_mead_historical <- function(){
   # -- from Alan Butler's code --
   hData <- read.csv('src_data/MeadHistorical.csv', stringsAsFactors = F)
   # add RunType column to historical data
   hData$RunType <- 99
-  modData <- read.csv('src_data/Mead24MSResults.csv', stringsAsFactors = F)
-  meadData <- rbind(hData, modData)
-  
-  # limit to only Mead, Pool Elevation
-  meadData <- dplyr::filter(meadData, Object.Name == 'Mead', Slot.Name == 'Pool Elevation')
+  meadData <- dplyr::filter(hData, Object.Name == 'Mead', Slot.Name == 'Pool Elevation')
   
   meadData$posDate = as.POSIXct(meadData$Timestep, format='%Y/%m/%d')
-  
   return(meadData[sort.int(meadData$posDate, index.return=TRUE)$ix, ])
+}
+
+read_mead_historical <- function(){
   
+  data = read.csv('src_data/DynamicMeadHistorical.csv', stringsAsFactors = F)
+  data$posDate = as.POSIXct(data$posDate, format='%Y-%m-%d')
+  return(data)
+}
+
+get_mead_filename <- function(){
+  date <- Sys.time()-86400*14 # two weeks earlier
```

Comment:
Couldn't this be used to recreate the "Most Probable" month, or is that a stretch on our part?